llpc/util/llpcFile.cpp

```diff
@@ -110,13 +110,8 @@ Result File::open(const char *filename, unsigned accessFlags) {
     }
 
     if (result == Result::Success) {
-#if defined(_WIN32)
-      // MS compilers provide fopen_s, which is supposedly "safer" than traditional fopen.
-      fopen_s(&m_fileHandle, filename, &fileMode[0]);
-#else
       // Just use the traditional fopen.
```

Comment:
Do we need this comment since the other alternative is gone?

Reply:
No